packages/svelte/src/internal/client/dom/elements/bindings/select.js

```diff
@@ -124,14 +137,12 @@ export function bind_select_value(select, get, set = get) {
 }
 
 /**
- * @template V
  * @param {HTMLSelectElement} select
- * @param {V} value
+ * @param {unknown[]} value
```

Comment:
Why this change?

Reply:
It doesn't make much sense to have a templated function that does not use nor restrict its type parameter, it can be replaced with `any`. Asking for `unknown[]` makes it more type-safe, `// @ts-ignore` is no longer required.